src/Scraping.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+
+from pyquery import PyQuery
+from urllib.request import Request
+from urllib.request import urlopen
+from urllib.parse import quote_plus
+
+# Reminder: quote_plus is used for properly quoting 
+# when building up a query string that go into a URL
+# https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote_plus
+
+# Google URL
+google_url = "https://www.google.it/search?q="
+# User agent for http get
+headers = {'User-Agent':'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36'}
+
+def define_url(question_text, answer_text=None):
+    if answer_text:
```

Comment:
non preferiresti una cosa tipo
```suggestion
   url = quote_plus(f"{question_text} {answer_text}") if answer_text else quote_plus(question_text)
   return google_url + url
```